libglusterfs/src/glusterfs/globals.h

```diff
@@ -124,6 +124,8 @@
 
 #define GD_OP_VERSION_9_0 90000 /* Op-version for GlusterFS 9.0 */
 
+#define GD_OP_VERSION_9_0 100000 /* Op-version for GlusterFS 10.0 */
```

Comment:
The Macro name should get changed to 10_0 and not 9_0 (it is repeat declaration.